### PR TITLE
[Feature:Developer] Add human names for CI checks

### PR DESCRIPTION
**Pull request by powe97**
### What is the current behavior?
Currently many of the CI checks are named `python-unit`, `css-lint`, `ansible-install`, etc.

### What is the new behavior?
This PR gives them cleaner names, "Python Unit," "CSS Lint," "Ansible Install," etc.

### CODE REVIEW

**Review by zacharymnp — APPROVED** [review]
This seems reasonable. My only concern is that submitty_test aliases were using names this PR is changing.

File: .github/workflows/submitty_ci.yml
Comment (IDzyre) [inline .github/workflows/submitty_ci.yml:90]: To me adding 'test' or 'tests' to the end of the name feels kind of redundant, but that is debatable. 

Should we capitalize the first letter of each word? It seems like it would look a bit better to me, but again it's debatable.

**Review by IDzyre — CHANGES_REQUESTED** [review]

File: .github/workflows/submitty_ci.yml
Comment (powe97) [reply .github/workflows/submitty_ci.yml:90]: I suppose "test(s)" is a bit redundant but generally when I refer to these I say "unit test(s)" rather than just "unit," whereas I would say just "lint."

As for capitalization, I'm indifferent.

**Comment by sahilsuman933** [comment]
LGTM

**Review by IDzyre — APPROVED** [review]

File: .github/workflows/submitty_ci.yml
Comment (williamjallen) [inline .github/workflows/submitty_ci.yml:117]:
```suggestion
    name: PHP Lint
```

**Review by williamjallen — COMMENTED** [review]
One comment.  Otherwise, LGTM.

File: .github/workflows/submitty_ci.yml
Comment (ziesski) [reply .github/workflows/submitty_ci.yml:117]: If we are fixing this, I think we also should fix python.

File: .github/workflows/submitty_ci.yml
Comment (ziesski) [inline .github/workflows/submitty_ci.yml:187]:
```suggestion
    name: Python Lint
```

**Review by ziesski — APPROVED** [review]

File: .github/workflows/submitty_ci.yml
Comment (williamjallen) [reply .github/workflows/submitty_ci.yml:117]: Yep, good catch.  I missed that one.

**Review by williamjallen — COMMENTED** [review]
Should these ones get a new name too?  
image

**Comment by powe97** [comment]
@williamjallen Sure, I missed those.

File: .github/workflows/db_check.yml
Comment (williamjallen) [inline .github/workflows/db_check.yml:1]: Is there a reason why this check isn't part of the regular CI workflow?  I don't really see a need for this to only run on PRs which touch migrations, which I believe is the only reason why this is in a separate file.

File: .github/workflows/pr_title_check.yml
Comment (williamjallen) [inline .github/workflows/pr_title_check.yml:1]: What do you think about renaming this to "Documentation"?  We may want to add more PR documentation checks in the future, and this workflow would be a good place to put them.

File: .github/workflows/submitty_ci.yml
Comment (williamjallen) [inline .github/workflows/submitty_ci.yml:1]: Should this file be renamed to just `ci.yml`?

**Review by williamjallen — CHANGES_REQUESTED** [review]

File: .github/workflows/db_check.yml
Comment (powe97) [reply .github/workflows/db_check.yml:1]: I'm not sure why it's like that. We can change it if desired.

File: .github/workflows/pr_title_check.yml
Comment (powe97) [reply .github/workflows/pr_title_check.yml:1]: That sounds good!

File: .github/workflows/submitty_ci.yml
Comment (powe97) [reply .github/workflows/submitty_ci.yml:1]: Sure, I'll do that.

File: .github/workflows/db_check.yml
Comment (powe97) [reply .github/workflows/db_check.yml]: Moved it into ci.yml.

**Review by williamjallen — COMMENTED** [review]
Should `ShellCheck` be renamed to `Shell Lint` for consistency with the other names?  It seems like I have new ideas every time I look at this, but I think this is my last suggestion.

Also: please make sure you are using the review system properly.  If you're waiting for a review, the PR should be placed in the "In Review" or "Awaiting Maintainer Review" columns.

**Review by williamjallen — APPROVED** [review]

**Pull request merged**
